core/domain/exp_services_test.py

```diff
@@ -48,6 +48,15 @@
 
 
 def _count_at_least_editable_exploration_summaries(user_id):
+    """Counts exp summaries that are at least editable by the given user.
+
+    Args:
+        user_id: unicode. The id of the given user.
+
+    Returns:
+        int. number of exp summaries that are at least editable by the
```

Comment:
Change to -> The number of exploration summaries that are at least editable by the given user.